package.json

```diff
@@ -10,7 +10,7 @@
     "url": "https://github.com/hypermodules/sitedown/issues"
   },
   "dependencies": {
-    "cheerio": "^0.22.0",
+    "cheerio": "^1.0.0-rc.3",
```

Comment:
might be better to wait until final cheerio 1.x release but if you need this and it's blocking you from doing something else i won't stop you!

Reply:
Figured the same until I noticed that rc 3 was released 2 years ago and is on the latest npm channel.  Just trying to play catch up on transient warnings.  

I'll land this in a major, but keeping it separate will make it easy to revert for whatever reason if its determined the old version was better.